src/sentry/search/django/backend.py

```diff
@@ -289,20 +293,26 @@ def query(self, project, tags=None, environment=None, sort_by='date', limit=100,
         # This is a punt because the SnubaSearchBackend (a subclass) shares so much that it
         # seemed better to handle all the shared initialization and then handoff to the
         # actual backend.
-        return self._query(project, retention_window_start, group_queryset, tags,
+        return self._query(projects, retention_window_start, group_queryset, tags,
                            environment, sort_by, limit, cursor, count_hits,
                            paginator_options, **parameters)
 
-    def _query(self, project, retention_window_start, group_queryset, tags, environment,
+    def _query(self, projects, retention_window_start, group_queryset, tags, environment,
                sort_by, limit, cursor, count_hits, paginator_options, **parameters):
 
         from sentry.models import (Group, Environment, Event, GroupEnvironment, Release)
 
+        # this backend only supports search within one project
+        if len(projects) != 1:
+            raise NotImplementedError
```

Comment:
not sure if there's a better way to handle this...i just didn't want to go down the rabbit hole of updating a bunch of tagstore methods if in the long run, we aren't going to be able support this for the django backend because of performance concerns